Docker.mk

```diff
@@ -1,4 +1,4 @@
-XCP_NG_VER := 8.2
+XCP_NG_VER := 8.2-uefistored
```

Comment:
I'd keep 8.2 as the version and add -uefistored in the next line.

Reply:
Indeed!